cogs/tibia.py

```diff
@@ -221,6 +225,15 @@ def __init__(self, bot: NabBot):
         except CannotPaginate as e:
             await ctx.send(e)
 
+    @staticmethod
+    def _get_cached_user_(self, user_id, user_unknown, users_cache, user_servers):
```

Comment:
In a cog, all commands are listed first in alphabetical order, followed by auxiliary functions, this should be moved accordingly.